recipe/meta.yaml

```diff
@@ -24,19 +25,25 @@ requirements:
     - {{ compiler('rust') }}
     - cargo-bundle-licenses
   host:
-    - maturin >=0.13,<0.14
+    - maturin =0.13.7
     - pip
     - python
+    - wheel 
   run:
     - python
 
 test:
+  source_files:
```

Comment:
Is this required?

Reply:
It appears to be, but I'm not clear why.

Reply:
Likely because the tests require more than the `tests` directory. Which error did you see?

Reply:
No tests get run w/o that line.

Reply:
Ah, I see where the miscommunication is. Yes, `source_files` is required to run tests, but the value is probably excessive.

```
source_files:
  - .
 ```

copies the entire source code into the test environment. What you may want to try is to replace it with `tests` (or whatever the directory name is). If they run doc tests, you can either skip those or add `docs` to the list as well.